src/current/cockroachcloud/cockroachdb-dedicated-on-azure.md

```diff
@@ -49,82 +49,84 @@ During [limited access](https://www.cockroachlabs.com/docs/{{ site.current_cloud
 
-### Is it possible to horizontally scale a dedicated cluster on Azure?
+### Is horizontal scaling supported?
 
 Not yet.
 
-### What Azure regions can we choose to create the dedicated clusters?
+### What Azure regions can we choose?
 
 You can create a cluster in `eastus2` (US East Coast - Virginia) or `westeurope` (Netherlands) during the limited access period. Contact your account team to express interest in other regions.
 
-### What kind of compute and storage resources are used for the dedicated clusters on Azure?
+### What kind of compute and storage resources are used?
 
-CockroachDB {{ site.data.products.dedicated }} clusters on Azure use [Dasv5-series VMs](https://learn.microsoft.com/azure/virtual-machines/dasv5-dadsv5-series) and [Premium SSDs](https://learn.microsoft.com/azure/virtual-machines/disks-types#premium-ssds). This configuration was selected for its optimum price-performance ratio after thorough performance testing across VM families and storage types.
+{{ site.data.products.dedicated }} clusters on Azure use [Dsv4-series VMs](https://learn.microsoft.com/en-us/azure/virtual-machines/dv4-dsv4-series) and [Premium SSDs](https://learn.microsoft.com/azure/virtual-machines/disks-types#premium-ssds). This configuration was selected for its optimum price-performance ratio after thorough performance testing across VM families and storage types.
 
-### We use CockroachDB {{ site.data.products.cloud }} credits to pay for our usage on AWS or GCP. Is it possible to use those same credits for CockroachDB {{ site.data.products.dedicated }} clusters on Azure?
+### Can we use {{ site.data.products.db }} credits to pay for clusters on Azure?
 
 Yes, existing CockroachDB {{ site.data.products.cloud }} customers can pay for the usage of CockroachDB {{ site.data.products.dedicated }} clusters on Azure with their available credits. To add additional credits to your CockroachDB {{ site.data.products.cloud }} organization, contact your Cockroach Labs account team.
 
-### Does the CockroachDB {{ site.data.products.cloud }} technical SLA cover the dedicated clusters on Azure?
+### Does the CockroachDB {{ site.data.products.cloud }} technical SLA cover clusters on Azure?
 
 During [limited access](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/cockroachdb-feature-availability), the [CockroachDB {{ site.data.products.cloud }} technical SLA](https://cockroachlabs.com/cloud-terms-and-conditions/cockroachcloud-technical-service-level-agreement/) does not apply to CockroachDB {{ site.data.products.dedicated }} clusters on Azure. For more details about the roadmap, contact your Cockroach Labs account team.
 
-### Are backups available for CockroachDB {{ site.data.products.dedicated }} clusters on Azure? Can we take our own backups to Azure storage in our tenant?
+### Are backups available for clusters on Azure? Can we take our own backups to Azure storage in our tenant?
```

Comment:
CC-managed backups are also available for public preview clusters, now.

Reply:
@ashleykwtam is handling this in https://cockroachlabs.atlassian.net/browse/DOC-7935.

Reply:
I've grabbed this after all.